pkg/activator/handler/handler.go

```diff
@@ -212,6 +207,8 @@ func (a *activationHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			logger.Errorw("Error processing request in the activator", zap.Error(err))
 		}
 	}
+	// This is right after WriteHeader, which intends to be used internally
+	w.Header().Set(activator.ProxyAttempts, strconv.Itoa(attempts))
```

Comment:
If `WriteHeader` happened, setting headers won't work, but I don't think that matters, especially that `ProxyAttempts` does not relate to latency and can be reported from here.

Reply:
@vagababov the header would not work since it will not appear in the response header, but the requestmetrichandler still can get it. report the metrics in two different places, will it be a little strange then?

Reply:
We used to have this header abusing before and I loved when we got rid of it. I don't think it's weird to have reporting in different places. I think currently the Report calls are a bit monolithic and we'll need to break them apart. Reporting metrics where they are generated seems generally nicer from my PoV though.

Reply:
done